src/main/java/pl/sdacademy/pewex/product/controller/ControlPanelController.java

```diff
@@ -0,0 +1,33 @@
+package pl.sdacademy.pewex.product.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.sdacademy.pewex.product.model.Category;
+import pl.sdacademy.pewex.product.service.CategoryService;
+
+@Controller
+@RequestMapping("/api/controlPanel")
+public class ControlPanelController {
+
+
+    private final CategoryService categoryService;
+
+    public ControlPanelController(CategoryService categoryService) {
+        this.categoryService = categoryService;
+    }
+
+    @GetMapping("/addCategory")
```

Comment:
addCategory sugeruję zmianę a metoda Get jest do czytania - nie pasuje to do siebie

Reply:
zmiana na Rest - zostało usunięte